karate-core/src/main/java/com/intuit/karate/ScriptValue.java

```diff
@@ -265,6 +265,10 @@ public ScriptValue invokeFunction(ScriptContext context) {
     }
 
     public String getAsPrettyString() {
+        return getAsPrettyString(true);
+    }
+
+    public String getAsPrettyString(boolean escapeJSON) {
```

Comment:
I'd like to understand why this change is needed

Reply:
For a request with body like `And request { "id": "http://example.com?id=1", "name": "Billie" }` getAsPrettyString method return escaped JSON and escapes `//` like `\/\/`. New method allows to get just pretty printed JSON not escaped. Please see the screen-shot below how the request looks in log and tooltip in vars window but in raw request response pane it shows just what user passed. 
![image](https://user-images.githubusercontent.com/24655564/36076111-ffcd3802-0f4f-11e8-9ae2-7b9e582e13fa.png)

Reply:
ok, it may be a mistake in the way we were doing pretty printing. if you find a better way to escape JSON values (for e.g. if there is a double-quote) what happens, if your approach takes care of this, then we can switch to that for everything

Reply:
Ahh, I can see the actual problem now. Will try to sort it out tomorrow.

Reply:
I have reverted the change in "getAsPrettyString" method. Will raise a separate issue for above problem and will resolve it in separate PR.